index.html

```diff
@@ -8,32 +8,201 @@
-    <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">Glad to see <br> you!</span></h1>
-    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
-        work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-        contact me.</p>
-    
-        <div class="connect">
-            Let's connect
+    <!-- first page -->
+    <section class="cover">
+        <div class="header_margin">
+            <header id="header">
+                <h1 class="logo"><a href="">My Logo</a></h1>
+                <nav class="nav_icon">
+                    <a href="#"><img src="./images/menu-icon.png" alt="menu-icon"></a>
+                </nav>
+            </header>
+            <section class="page-one" id="first_page">
+                <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">Glad to see <br> you!</span></h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                <div class="connect">
+                    Let's connect.
+                </div>
+                <div class="links">
+                    <ul>
+                        <li><a href="#"><img src="./images/twitter.svg" alt="twiter_icon"></a></li>
+                        <li><a href="#"><img src="./images/linkedin.svg" alt="linkedin_icon"></a></li>
+                        <li><a href="#"><img src ="./images/medium.svg" alt="medium_icon"></a></li>
+                        <li><a href="#"><img src="./images/github.svg" alt="github_icon"></a></li>
+                        <li><a href="#"><img src="./images/angellist.svg" alt="angellist_icon"></a></li>
+                    </ul>
+                </div>
+            </section>
+        </div>
+    </section>
+
+    <section id="mid">
+        <!-- second page -->
+         <div class="left-block">
+            <div class="primary-image">
+                <img src="images/project1.png" alt="photo_of_my_projects">
+            </div>
+            <div class="title">
+                <h1>Tonic</h1>
+            </div>
+            <div class="frame-2">
+                <ul>
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="primary-text"> Daily selection of privately personalised reads" no accounts or signups required. </div>
+            <div class="tags"> 
+                <ul>
+                    <li>html</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <div class="action">See project</div>
```

Comment:
- [ ] It is a link `a` or `button` tag, and please follow the button styling: Active, Hover and Pressed (see design)👇
![button](https://user-images.githubusercontent.com/66992901/193404080-3fedfa1b-ed9c-4879-a0f0-000bc3101cb8.PNG)
#### Please apply this for all your buttons🙏